docs/project/goals.md

```diff
@@ -506,15 +506,15 @@ interfaces with business logic in Carbon, although this direction can tolerate
 slightly more constraints both in supported features and performance overhead.
 In all cases, the particular performance overhead imposed by moving between C++
 and Carbon will need to be easily exposed and understood by developers. While a
-given piece code only needs to be migrated once, we expect interoperability to
+given piece of code only needs to be migrated once, we expect interoperability to
 be invoked continuously to support migrated code and will thus remain important
 for most developers.
 
 ## Non-goals
 
 There are common or expected goals of many programming languages that we
 explicitly call out as non-goals for Carbon. That doesn't make these things bad
-in any way, but reflects the fact that they do not provide meaningful value to
+in any way but reflects the fact that they do not provide meaningful value to
```

Comment:
```suggestion
in any way, but instead reflects the fact that they do not provide meaningful value to
```

We can increase contrast, but note:
https://www.grammarly.com/blog/comma-before-but/#:~:text=When%20Do%20You%20Need%20a,have%20to%20trip%20you%20up.